### DISCUSSION TITLE
MacOS: Swap `Command` with `Option` and vice versa

### DISCUSSION THREAD

**Discussion by joshua-light** [kovidgoyal/kitty#3745] [Q&A]
Hi!

Is there any way to easily swap `Command` key binding to `Option` and vice versa? I have a lot of bindings on `Alt` in apps like Vim. Luckily, Kitty allows to use `Option` as `Alt`. However, from the keyboard layout perspective, it'd be much more natural to use `Command` as `Alt`. Is there any way to do so?

Thanks in advance!

**Comment by kovidgoyal** [comment]
No, there's no easy way to do so. You could conceivably use https://sw.kovidgoyal.net/kitty/conf.html#shortcut-kitty.Send-arbitrary-text-on-key-presses to do what you want, but it wont be easy.

**Comment by basbebe** [comment]
You could use [Karabiner-Elements](https://karabiner-elements.pqrs.org/) or [skhd](https://github.com/koekeishiya/skhd) to do that
